msgs/yarp/thrift/CMakeLists.txt

```diff
@@ -44,12 +47,16 @@ set(LIBRARY_TARGET_NAME HumanDynamicsMsg)
 
 
 yarp_idl_to_dir(HumanDynamicsEstimation/HumanDynamics.thrift
-  ${CMAKE_CURRENT_BINARY_DIR}/autogenerated
+  OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR}/autogenerated
   SOURCES_VAR ${LIBRARY_TARGET_NAME}_GEN_SRC
   HEADERS_VAR ${LIBRARY_TARGET_NAME}_GEN_HDR
-  INCLUDE_DIRS_VAR ${LIBRARY_TARGET_NAME}_INCLUDE_DIRS)
+  INCLUDE_DIRS_VAR ${LIBRARY_TARGET_NAME}_INCLUDE_DIRS
+  CMAKE_SCRIPTS_VAR ${LIBRARY_TARGET_NAME}_CMAKE_SCRIPTS)
+foreach(_script ${YARP_dev_idl_CMAKE_SCRIPTS})
+  include("${_script}")
+endforeach()
 
-include_directories(${LIBRARY_TARGET_NAME}_INCLUDE_DIRS)
+# include_directories(${LIBRARY_TARGET_NAME}_INCLUDE_DIRS)
```

Comment:
cleanup

Reply:
cleaned in https://github.com/robotology/human-dynamics-estimation/pull/246/commits/0f514f17a9da8a12f23a68fdadb310742a20f769